lib/chef/knife/winrm.rb

```diff
@@ -49,7 +49,12 @@ def run
         STDOUT.sync = STDERR.sync = true
 
         configure_session
-        execute_remote_command
+        exit_status = execute_remote_command
+        if exit_status != 0
+          exit exit_status
```

Comment:
If this happens in a parallel bootstrap, I believe this would interrupt other, ongoing bootstraps.